thoth/adviser/steps/tensorflow_21_urllib3.py

```diff
@@ -40,25 +42,23 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+@attr.s(slots=True)
 class TensorFlow21Urllib3Step(Step):
     """A step that suggests not to use TensorFlow 2.1 with specific urllib3 versions that cause six import issues."""
 
+    _message_logged = attr.ib(type=bool, default=False, init=False)
+
     # Run this step each time, regardless of when TensorFlow and urllib3 are resolved.
     MULTI_PACKAGE_RESOLUTIONS = True
 
-    _SCORE_ADDITION = -0.8
-    _JUSTIFICATION_ADDITION = [
-        {
-            "type": "WARNING",
-            "message": (
-                "TensorFlow in version 2.1 can cause runtime errors when imported, caused by incompatibility between urllib3 and six packages"
-            ),
-            "link": jl("tf_21_urllib3"),
-        }
-    ]
-
+    _MESSAGE = f"TensorFlow in version 2.1 can cause runtime errors when imported, caused by " \
```

Comment:
TensorFlow in version 2.1 can cause runtime errors when imported. They are caused by...